TestBean/src/main/java/org/example/bean/RandomValueBeanPostProcessor.java

```diff
@@ -0,0 +1,39 @@
+package org.example.bean;
+
+import org.example.annotation.RandomValue;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Field;
+import java.util.Random;
+
+@Component
+public class RandomValueBeanPostProcessor implements BeanPostProcessor {
+    private final Random random = new Random();
+
+    @Override
+    public Object postProcessBeforeInitialization(Object bean, String beanName) throws BeansException {
+        return bean;
+    }
+
+    @Override
+    public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
+        Field[] fields = bean.getClass().getDeclaredFields();
```

Comment:
add check that you obtain needed bean, currently you process it in every bean instead of needed